react/Product.js

```diff
@@ -93,8 +116,8 @@ const composeAggregateOffer = (product, currency) => {
 
   const aggregateOffer = {
     '@type': 'AggregateOffer',
-    lowPrice: getSpotPrice(low),
-    highPrice: getPrice(high),
+    lowPrice: pricesWithTax ? Math.round(((getSpotPrice(low) + getTax(low)) + Number.EPSILON) * Math.pow(10, decimals)) / Math.pow(10, decimals) : getSpotPrice(low),
+    highPrice: pricesWithTax ? Math.round(((getPrice(high) + getTax(high)) + Number.EPSILON) * Math.pow(10, decimals)) / Math.pow(10, decimals) : getPrice(high),
```

Comment:
You could have a function like
```
const getFinalPrice = (value, decimals, getPriceFunc) => {
  return pricesWithTax ? Math.round(((getPriceFunc(value) + getTax(value)) + Number.EPSILON) * Math.pow(10, decimals)) / Math.pow(10, decimals) : getPriceFunc(value)
}
```

And then it'd be simplified to:
```suggestion
    lowPrice: getFinalPrice(low, decimals, getSpotPrice),
    highPrice: getFinalPrice(high, decimals, getPrice),
```
What do you think about it?

Reply:
That's a nice change to make the code readable as well as avoid duplicating it.  :) Made this change as well.  As for the get settings query, I thought I could do it with https://github.com/vtex-apps/render-runtime#getsettings but when I used vtex.structured-data instead of vtex.store it returned null. Either I'm reading the docs wrong or it just works with the vtex.store app which returns store related settings that are of no use to me in this case.  Let me know if you know of a better way to get the app's settings though, thanks.